service/models.py

```diff
@@ -205,3 +206,12 @@ def deserialize(self, data: dict) -> None:
             raise DataValidationError(
                 f"Invalid {type(self).__name__}: failed to deserialize request body\nError message: {error}"
             ) from error
+        
+    @classmethod
+    def get_dp_id(cls, shopcart_id, item_id):
```

Comment:
would you like to reuse get_by_id()? or add corresponding test case for this? lol

Reply:
Yes, I forgot to delete some part that didn't use. I will check the code again before merge

Reply:
no problem! thank you~~